metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -247,10 +247,10 @@ def validate_shares(
 
         if shares:
             # Check: platform_instance should be present
-            assert current_platform_instance is not None, (
-                "Did you forget to set `platform_instance` for current ingestion ? "
-                "It is required to use `platform_instance` when ingesting from multiple snowflake accounts."
-            )
+            if current_platform_instance is None:
+                logger.warning(
```

Comment:
I'd make this an INFO, and perhaps explain what setting platform instance provides (i.e. no conflict for entities with the same name in each instance). I think there are valid use cases for not setting platform instance, e.g. you don't expect overlap or you're looking at different slices of the same data

Reply:
done